ouroboros-consensus-cardano/test/Test/ThreadNet/Cardano.hs

```diff
@@ -361,36 +362,40 @@ prop_simple_cardano_convergence TestSetup
   , setupTestConfig
   , setupVersion
   } =
-    tabulate "ReachesShelley label" [label_ReachesShelley reachesShelley] $
-    tabulate "Observed forge during a non-overlay Shelley slot"
-      [label_hadActiveNonOverlaySlots] $
-    tabulatePartitionDuration $
-    tabulateFinalIntersectionDepth $
-    tabulatePartitionPosition $
-    prop_general_semisync PropGeneralArgs
-      { pgaBlockProperty       = const $ property True
-      , pgaCountTxs            = fromIntegral . length . extractTxs
-      , pgaExpectedCannotForge = noExpectedCannotForges
-      , pgaFirstBlockNo        = 0
-      , pgaFixedMaxForkLength  = Just maxForkLength
-      , pgaFixedSchedule       =
-          -- the leader schedule isn't fixed because the Shelley leader
-          -- schedule is (at least ideally) unpredictable
-          Nothing
-      , pgaSecurityParam       = setupK
-      , pgaTestConfig          = setupTestConfig
-      , pgaTestConfigB         = testConfigB
-      }
-      testOutput .&&.
+    prop_general_semisync pga testOutput .&&.
     prop_inSync testOutput .&&.
-    prop_ReachesShelley reachesShelley
+    prop_ReachesShelley reachesShelley .&&.
+    prop_noCPViolation .&&.
+    ( tabulate "ReachesShelley label" [label_ReachesShelley reachesShelley] $
+      tabulate "Observed forge during a non-overlay Shelley slot"
+        [label_hadActiveNonOverlaySlots] $
+      tabulatePartitionDuration $
+      tabulateFinalIntersectionDepth $
+      tabulatePartitionPosition $
+      property True
+    )
   where
     TestConfig
       { initSeed
       , numCoreNodes
       , numSlots
       } = setupTestConfig
 
+    pga = PropGeneralArgs
+        { pgaBlockProperty       = const $ property True
+        , pgaCountTxs            = fromIntegral . length . extractTxs
+        , pgaExpectedCannotForge = noExpectedCannotForges
+        , pgaFirstBlockNo        = 1
```

Comment:
This was 0, but should have been 1. These tests involve so many blocks that it doesn't really matter, but 1 is correct.